app/src/main/java/com/itssuryansh/taaveez/Edit_Content.kt

```diff
@@ -283,4 +312,74 @@ class Edit_Content : AppCompatActivity() {
         BackData()
     }
 
+    private fun updateLabels() {
+        val dialogView = LayoutInflater.from(this).inflate(R.layout.diaglog_insert_label, null)
```

Comment:
It is necessary, let's say if you click outside dialog box, then all the labels will disappear whereas they are stored in selectedLabels, because in the logic we are not making Chip of the already existing label, that's why I think it is correct

Reply:
I meant only doing the creation of dialog, setup of listeners and all other things which need not be done each and every time, only once